interface-definitions/vpn_ipsec.xml.in

```diff
@@ -99,6 +99,26 @@
                   </constraint>
                 </properties>
               </leafNode>
+              <leafNode name="rekey">
+                <properties>
+                  <help>Controls local-initiated rekey behavior</help>
+                  <completionHelp>
+                    <list>enable disable</list>
+                  </completionHelp>
+                  <valueHelp>
+                    <format>enable</format>
+                    <description>Initiate a re-key of the Security Association when approaching lifetime limits</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>disable</format>
```

Comment:
We agree to not use boolean values like yes/no true/false enable/disable 1/0, and so on
For example, if `mobike` is enabled by default, we use the `disable-mobike` option to avoid enable/disable

Reply:
Updated to fix.